src/portable_endian.h

```diff
@@ -54,6 +54,7 @@
 
 #	include <sys/endian.h>
 
+#if !defined(__FreeBSD__)
```

Comment:
Would #ifndef be more readable?

Reply:
I don't have a strong opinion about this. 

I used that construction because it looks more homogeneous for that particular file and it's been used in line 15 for instance.